static/app/views/replays/replayTable/index.tsx

```diff
@@ -28,6 +31,8 @@ import {
 import {ReplayColumn} from 'sentry/views/replays/replayTable/types';
 import type {ReplayListRecord} from 'sentry/views/replays/types';
 
+export const MIN_DEAD_RAGE_CLICK_SDK = '7.60.1';
```

Comment:
Is this imported somewhere?

Reply:
was planning to originally but not anymore -- i'll get rid of it